pom.xml

```diff
@@ -20,8 +20,8 @@
     </modules>
 
     <properties>
-        <tycho-version>1.4.0</tycho-version>
-        <tycho-extras-version>1.4.0</tycho-extras-version>
+        <tycho-version>1.5.1</tycho-version>
+        <tycho-extras-version>1.5.1</tycho-extras-version>
```

Comment:
According to maven it goes to 1.6 . Any reason to stop at 1.5 ?

Reply:
Yes, Tycho 1.6.0 no longer supports Java 12. https://wiki.eclipse.org/Tycho/Release_Notes/1.6#Drop_support_for_obsolete_modular_JVMs_.2810_and_12.29

Reply:
I think it make sense to upgrade Travis to use openjdk13 , 
plan is to verify build on LTS versions (8, 11) and whatever latest public version of java.

@Bananeweizen , please try to update travis config, if it works latest jdk and latest tycho ... good.